### ISSUE TITLE
Support named demos

### ISSUE THREAD

**Issue by rictic** [Polymer/polymer-analyzer#626]
e.g. https://github.com/PolymerElements/app-layout/blob/master/app-drawer/app-drawer.html#L50

We should also do a spot check of the analyzer against app-layout afterwards to be sure we're getting the demos right.

**Comment by usergenic** [comment]
@samuelli @rictic Other than the fact that demos weren't coming in from `@demo` annotations in html comments, does this bug mean you want to change the `AnalysisFormat` to emit the `demos` as something other than `string[]`, i.e. `{path: string, description: string}[]`?

**Comment by samuelli** [comment]
Yep. This is to match hydrolysis behaviour so demos can have nicer titles.

**Comment by samuelli** [comment]
Is it possible get a release after #633?

**Comment by rictic** [comment]
@samuelli Yeah definitely, and I'm reviewing it now. Sorry, yesterday was consumed by google3 stuff.

**Comment by samuelli** [comment]
I'd like to push out our analyzer support the next couple of days and this is our last blocking issue. Any update @usergenic?

**Comment by usergenic** [comment]
@sameulli I have a new PR out for this feature as of this morning https://github.com/Polymer/polymer-analyzer/pull/658

Ping reviewers at your discretion.

**Comment by samuelli** [comment]
Awesome, thanks!

**Comment by usergenic** [comment]
#658 has been merged into master, @samuelli so closing this issue.  We'll hopefully release a new cut of Analyzer supersoon!
